src/Halogen/Svg/Attributes.purs

```diff
@@ -288,7 +300,7 @@ strokeLineJoin = attr (H.AttrName "stroke-linejoin") <<< printStrokeLineJoin
 strokeMiterLimit :: forall r i. Number -> IProp (strokeMiterLimit :: String | r) i
 strokeMiterLimit = attr (H.AttrName "stroke-miterlimit") <<< show <<< max 1.0
 
-strokeOpacity :: forall r i. Number -> IProp (strokeOpacity :: String | r) i
+strokeOpacity :: forall r i. Number -> IProp (strokeOpacity :: Number | r) i
```

Comment:
Why are you making this change?

Reply:
Split off into https://github.com/JordanMartinez/purescript-halogen-svg-elems/pull/40 rather than sneaking in with this